system.go

```diff
@@ -43,3 +55,12 @@ func (g *Glutton) startMonitor(quit chan struct{}) {
 		}
 	}()
 }
+
+func isCommandAvailable(name string) bool {
+
+	cmd := exec.Command("/bin/sh", "-c", "command -v "+name)
+	if err := cmd.Run(); err != nil {
```

Comment:
Did you test that this returns an error for an unknown command?

Reply:
Yep tried and tested boss:)